content/swift/concepts/arrays/terms/shuffle/shuffle.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.shuffle()'
+Description: 'Randomly shuffles the elements of an array.'
+Subjects:
+  - 'Swift'
+  - 'Mobile Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'Randomization'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.shuffle()`** method in Swift Arrays is used to randomly shuffle the elements of an array. It reorders the elements in a random order, providing a different arrangement of the elements each time it is called.
+
+## Syntax
+
+The syntax of the **`.shuffle()`** method is as follows:
+
+```pseudo
+arrayName.shuffle()
+```
+
+The **`arrayName`** is the name of the array that you want to shuffle.
+
+## Example
+
+Consider an array of numbers:
+
+```swift
```

Comment:
Can we combine the two codeblocks (the array declaration and the shuffle example) into one. Rather than splitting the example code between two separate code blocks.